src/Components/ContactForm.jsx

```diff
@@ -43,97 +51,120 @@ const ContactForm = () => {
     from_name: "",
     reply_to: "",
     contact_type: "",
-    message: ""
-  })
+    message: "",
+  });
+
+  const [emailError, setEmailError] = useState({
+    errorStatus: false,
+    errorMessage: "",
+  });
+
+  const validateEmail = (email) => {
+    const validEmail = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
Can you add a comment linking to the page where you got this from or describing what it does?

Reply:
Also this doesn't need to happen in this PR but it would be good to extract this to a helper util file so that it can be used elsewhere. And then add unit tests for it.

Reply:
@dylankershaw I added a comment explaining what it does and linking the site, as well as two TODOs: one to migrate to a util and unit test, and another where we may want to use a library and future proof ourselves for feature expansions where we want to deal with more form types and data validation. Let me know what you think.